2-mandatory/8-password-validator.js

```diff
@@ -23,7 +23,13 @@ PasswordValidationResult=  [false, false, false, false, true]
 
 */
 
-function validatePasswords(passwords) {}
+function validatePasswords(passwords) {
+   return passwords.map((item,index)=>{
+        if(containsLowercaseLetter(item) && containsUppercaseLetter(item) && containsNumber(item) && containsSymbol(item) && item.length >= 5 && passwords.indexOf(item)===index) return true;
+       else return false; 
+    }
+    )
+}
```

Comment:
I have used string methods to check the types, the example used regular expression literals